src/Application.js

```diff
@@ -1,14 +1,20 @@
-import { Footer, Navbar } from "./show/components"
-import { Home } from "./show/pages"
+import { Home, Standings } from './show/pages';
+import { Footer, Navbar } from './show/components';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
```

Comment:
React-router-dom should be the first, then empty line and the others. 

We group them by libraries imports, then in house stuff, and the these grouped imports are sorted by length and if the case alphabetically.